docs/docsite/rst/community/communication.rst

```diff
@@ -49,7 +49,7 @@ The Ansible community maintains its own Matrix homeserver at ``ansible.im``, how
 Matrix chat supports:
 
 * persistence (when you log on, you see all messages since you last logged off)
-* edits (so you can fix your typos)
+* edits (so you can fix your typos - **NOTE** that every edit on Matrix causes the message to be sent again to IRC, so please avoid editing multiple times!)
```

Comment:
```suggestion
* edits (Lets you fix typos and so on. **NOTE** Each edit you make on Matrix re-sends the message to IRC. Please try to avoid multiple edits!)
```

Reply:
Hope you don't mind suggestions. Feel free to take or leave. Cheers.